mmd_tools/core/model.py

```diff
@@ -813,7 +757,8 @@ def join_models(parent_root_object: bpy.types.Object, child_root_objects: Iterab
             except Exception:
                 logging.exception("Error removing child root object")
 
-        # Clean and reapply additional transformations to properly set up all bones and constraints
+        # Apply additional transform
+        bpy.context.view_layer.objects.active = parent_root_object
```

Comment:
I suggest using `FnContext.set_active_object` instead of `bpy.context.view_layer.objects.active`, because the Blender API related to `Context` is unstable and difficult to use.
Therefore, I recommend developing and enhancing `FnContext` if possible.